src/SHAutomation.Core/StaticClasses/SHSpinWait.cs

```diff
@@ -5,7 +5,7 @@ namespace SHAutomation.Core.StaticClasses
 {
     public static class SHSpinWait
     {
-      
+
 
         public static bool SpinUntil(Func<bool> condition, TimeSpan timeout)
```

Comment:
@ThomasPoucher Part of me thinks this should just call the new method below and just pass `None` as the `CancellationToken` param. Would be a big change as essentially abandoning the built in spin wait. What do you think?

Reply:
We could even rename from `SHSpinWait` to something else also

Reply:
Maybe `WaitForAction` ?